src/main/java/com/yas/service/CustomerService.java

```diff
@@ -1,27 +1,46 @@
 package com.yas.service;
 
+import java.util.List;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
 
 import com.yas.dao.CustomerRepository;
+import com.yas.dao.ServicesRepository;
 import com.yas.model.Customer;
+import com.yas.model.Services;
 
 @Service
 public class CustomerService {
-	
+
 	@Autowired
 	CustomerRepository cr;
-	
+
+	@Autowired
+	ServicesRepository sr;
+
 	public Customer getCustomer(Customer customer) {
-		Customer custom = cr.findByAll(customer.getFirstName(), 
-									   customer.getLastName(),
-									   customer.getAddress(),
-									   customer.getMobNo(),
-									   customer.getEmail());
+		Customer custom = cr.findByAll(customer.getFirstName(), customer.getLastName(), customer.getAddress(),
```

Comment:
check this logic again

Reply:
I have done unit testing please review it.